sdk/python/feast/infra/utils/postgres/postgres_config.py

```diff
@@ -6,6 +6,8 @@
 
 
 class PostgreSQLConfig(FeastConfigBaseModel):
+    min_conn: int = 1
+    max_conn: int = 10
```

Comment:
create a connection definition variable that will be used from `_get_conn()`

**e.g:**
```py
type_conn: str = "normal" # or "pool"
```